easypermissions/build.gradle

```diff
@@ -20,7 +20,12 @@ android {
 }
 
 dependencies {
-    compile "com.android.support:appcompat-v7:$support_library_version"
+    implementation "com.android.support:appcompat-v7:$support_library_version"
+    api "com.android.support:support-compat:$support_library_version"
+    api "com.android.support:support-fragment:$support_library_version"
```

Comment:
When I removed `appcompat-v7` from the demo project, I couldn't get it to compile without these api declarations.